socket.c

```diff
@@ -78,6 +79,13 @@ bool cSatipSocket::Open(const int portP, const bool reuseP)
      ERROR_IF_FUNC(setsockopt(socketDescM, SOL_IP, IP_PKTINFO, &yes, sizeof(yes)) < 0,
                    "setsockopt(IP_PKTINFO)", Close(), return false);
 #endif // __FreeBSD__
+
```

Comment:
Change the empty line to a comment line: ```// Tweak receive buffer size if requested```

Reply:
Done in latest commit.